ipfs-cluster-service/main.go

```diff
@@ -263,7 +263,15 @@ func run(c *cli.Context) error {
 		cfg.AllocationStrategy = a
 	}
 
-	api, err := restapi.NewRESTAPI(cfg.APIAddr)
+	var api *restapi.RESTAPI
+	if cfg.APIUseTLS {
```

Comment:
Ok, following from above, here you can check if the CertFile or KeyFile are defined. Attempt to create the TLS Config and then pass it to NewTLSRESTAPI.